src/app/containers/SocialEmbed/index.jsx

```diff
@@ -24,6 +24,7 @@ const logger = nodeLogger(__filename);
  * NB Tweets max-out at 500px, which is represented as 31.25rem.
  */
 const MAX_WIDTH = '31.25rem';
+const MIN_HEIGHT = 300;
```

Comment:
- We don't operate on this number, so it should probably include the unit (as above).
- This should probably be the `rem` equivalent of `300px`.